skins/cat17/src/scripts/donationForm.js

```diff
@@ -336,6 +336,12 @@ $( function () {
 		var nextRequired = currentState.donationFormContent.paymentType === 'BEZ' ? $( '#payment-method' ) : $( '#donation-type .legend:first' );
 		var $introBanner = $('.introduction-banner');
 		$introBanner.insertBefore( nextRequired ).removeClass( 'hidden' );
+
+		// On mobile, the state bar is hidden initially and only shown when scrolling, which leads to wrong offset.
+		// Thus, we have to make it visible for offset calculation
+		if ( $( window ).width() < 1023 ) {
```

Comment:
This could be a little less painful to read (and would spare us from hard-coding the 1023 again) if we just always added the active class without this check - the comment does a good job at pointing out what it's good for, and desktop does not seems to have a problem with the class added while the state-bar is hidden.